browser/src/map/handler/Map.Infobar.js

```diff
@@ -16,7 +16,19 @@ L.Map.Infobar = L.Handler.extend({
 
 		if (viewInfo && !this.enabled() && viewInfo.userextrainfo &&
 		    viewInfo.userextrainfo.is_admin) {
-			this.enable();
+			var laterDate = new Date();
+			var currentDate = new Date();
+			var timeValue = window.localStorage.getItem('InfoBarLaterDate');
+			if (!timeValue || isNaN(timeValue)) {
+				/* + 5 seconds */
+				laterDate.setTime(currentDate.getTime() + 5000);
```

Comment:
Maybe we need to make this -5000 not +5000 so
"if (currentDate > laterDate)" will work :))
Thanks for the fix :) we just need to change that "+" to "-" i think.